crates/interledger-packet/src/packet.rs

```diff
@@ -582,7 +598,7 @@ fn deserialize_envelope(
         return Err(PacketTypeError::Unexpected(got_type, packet_type as u8).into());
     }
 
-    let content_offset = 1 + {
+    let content_offset = PacketType::LEN + {
```

Comment:
I was thinking that adding these LEN's everywhere was in vain, but at least this deserialize_envelope is now more clear, in my opinion.